match/match.go

```diff
@@ -77,6 +78,12 @@ func defaultRouter(env *env) *mux.Router {
 	return r
 }
 
+// respondWithError responds to a HTTP request with a JSON error response
+func respondWithError(w http.ResponseWriter, err string, statusCode int, requestType string) {
+	w.WriteHeader(statusCode)
+	fmt.Fprintln(w, util.CreateErrorJSON(err))
```

Comment:
How come you're not adding the metric for failure here like in the other service?

Reply:
That was just so the PR size wouldn't balloon, since none of the metrics stuff is set up for the match service yet. *Coming soon* 😉